core/src/main/java/com/klaytn/caver/contract/ContractMethod.java

```diff
@@ -299,36 +509,100 @@ public ContractMethod(Caver caver, String type, String name, List<ContractIOType
      * @throws TransactionException
      */
     public TransactionReceipt.TransactionReceiptData sendWithSolidityWrapper(List<Type> wrapperArguments, SendOptions options, TransactionReceiptProcessor processor) throws IOException, TransactionException {
-        List<Type> functionParams = new ArrayList<>();
+        if(!getType().equals(TYPE_FUNCTION)) {
```

Comment:
Can you explain what this logic for?

Reply:
This logic was added to make this function available only when executing the contract's function.

Reply:
So if `getType().equals(TYPE_FUNCTION)` is `true`, does it mean that it is a function of a smart contract?

Reply:
It sets the data parsed from the 'type' field in the ABI's function information.